docs/notebooks/pulse-efficient-transpilation.ipynb

```diff
@@ -408,15 +470,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "From the plot above, it is clear that the `RZXGate` transpilation results in shorter circuit duration for small entangling angles\n",
+    "As shown in the plot above, the `RZXGate` transpilation results in shorter circuit durations for small entangling angles.\n",
     "\n",
     "### A larger example\n",
-    "Let's extend our example to a 1D spin chain of $N=21$ qubits. We follow the same steps to transpile into a basis gate set that includes either an `ecr` or a `rzx` gate. This is what the trotterized evolution circuit looks like."
+    "\n",
+    "Extend the example to a one-dimensional spin chain of $N=21$ qubits. Follow the previously described steps to transpile into a basis gate set that includes either an `ECR` or an `RZX` gate. This is what the Trotterized evolution circuit looks like:"
```

Comment:
is there any way to increase N even more? or is 21 the limit?

Reply:
Asking @haimeng-zhang about this.